src/lib/utils/referrerUtils.ts

```diff
@@ -1,5 +1,17 @@
 import logger from "../Logger";
 
+/**
+ *
+ * @param referrer - redirect url - the specific url prior to the url the user is currently on
```

Comment:
I would rather say something that informs that this is the url from which the user arrive at the current url, or something like that.